### PR TITLE
Add mdn_url et spec_url to SVGMarkerElement.json

### PR DESCRIPTION
**Pull request by teoli2003**
Update methods and spec for all SVGMarkerElement children (and the interface itself)

Note that SVGMarkerElement.orient is not documented on MDN.

### CODE REVIEW

File: api/SVGMarkerElement.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "api": {
     "SVGMarkerElement": {
       "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGMarkerElement",
+        "spec_url": "https://www.w3.org/TR/SVG2/painting.html#InterfaceSVGMarkerElement",
```
Comment (Elchi3) [inline api/SVGMarkerElement.json:6]:
```suggestion
        "spec_url": "https://svgwg.org/svg2-draft/painting.html#InterfaceSVGMarkerElement",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Please use https://svgwg.org/svg2-draft throughout which is the latest version

File: api/SVGMarkerElement.json (hunk 1 above)
Comment (teoli2003) [reply api/SVGMarkerElement.json:6]: No problem. Will do. 

The older was listed at browser-specs, though. Should (could?) we add a linter test that detect this?

Comment (Elchi3) [reply api/SVGMarkerElement.json]: > The older was listed at browser-specs, though. Should (could?) we add a linter test that detect this?

I will think about this more. I didn't want to start with forbidding everything, but maybe we can get more strict :) browser-specs has a concept of nightly specs that are preferable and we can investigate how much non-nightly versions we have in BCD by tweaking the linter: https://github.com/mdn/browser-compat-data/blob/main/test/spec-urls.test.js#L16

Comment (teoli2003) [reply api/SVGMarkerElement.json]: Thanks! Not urgent though. (I pushed the changes)

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**